components/dashboard/src/components/start-workspace.tsx

```diff
@@ -120,8 +120,8 @@ export class StartWorkspace extends React.Component<StartWorkspaceProps, StartWo
         }
 
         const defaultErrMessage = `Error while starting workspace ${workspaceId}`;
-        this.props.service.server.startWorkspace(workspaceId)
-            .then(workspaceStartedResult => {
+        this.props.service.server.startWorkspace(workspaceId, forceDefault)
+            .then((workspaceStartedResult: { instanceID: string; workspaceURL: string; }) => {
```

Comment:
any reason to write the type here?

Reply:
Yes, Theia shows it as an error.
`Parameter 'workspaceStartedResult' implicitly has an 'any' type.ts(7006)`